.release-please-manifest.json

```diff
@@ -2,5 +2,5 @@
   "actions": "9.0.2",
   "reusable-workflows": "11.3.0",
   "config-inject": "3.1.1",
-  "terraform-module": "3.1.1"
```

Comment:
Did you update this yourself? Usually this is going to be updated by a release bot once this PR gets merged.

Reply:
oh great. i'll revert. i thought it was manual